src/test/java/de/szut/lf8_project/FullIntegrationTest.java

```diff
@@ -168,6 +168,22 @@ protected Project createDefaultProject(final ProjectLeadId projectLeadId) {
         return project;
     }
 
+    protected Project createDifferentProject(final ProjectLeadId projectLeadId) {
```

Comment:
mh, also richtig holt mich die methode nicht ab, was ist das nicht, createAnotherDifferentProject? ;)

wird glaub ich nur 1x gecalled?

im Inte-Test wird ja auch (berechtigterweise) kein String literal geprüft. ob du in dem test 2x `createDefault` verwendest oder eben 2 separate Methoden kommt ja aufsselbe raus.

Reply:
'''

Reply:
damit kann ich leben :D